notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -2467,6 +2467,58 @@ computer hacking [Digicorp tried to "jack in" to the Sunway Systems computer sys
 resentment [Virgil had a chip on his shoulder on account that his job of figuring out when people were lying had recently been taken over by a machine.]
 
 
+movie: Signs (2002)
+===================
+
+:: Title
+Signs
+
+:: Description
+Signs is a 2002 American science fiction mystery thriller film written and
+directed by M. Night Shyamalan and produced by Shyamalan, Frank Marshall,
+Kathleen Kennedy and Sam Mercer.
+
+Synopsis: A former Episcopal priest discovers a series of crop circles in his
+cornfield, and slowly discovers that the phenomenon is a result of
+extraterrestrial life.
+
+:: Date
+2002-08-02
+
+:: References
+https://en.wikipedia.org/wiki/Signs_(2002_film)
+
+:: Ratings
+4 <paul>
+
+:: Major Themes
+extraterrestrial being [The story follows Graham and his family as they barricade themselves in their isolated farmhouse in the midst of an alien invasion of Earth.]
```

Comment:
Off topic motivation.

Reply:
Maybe I'm wrong but I think you didn't use this theme in Impostor. Yet there was an alien invasion in both cases.

Reply:
Happily reworked the motivation. But we have *extraterrestrial being* as major in Impostor (2002).